lib/model.dart

```diff
@@ -14,6 +14,16 @@ class Person {
   String location;
   List<String> themes;
   int messageCount;
+
+  Person(this.age, this.age_category, this.gender, this.IDPStatus,
+      this.location, this.themes, this.messageCount);
+
+  factory Person.fromObj(Map<String, dynamic> obj) {
```

Comment:
Where would I get one of these objects from? Is the expectation that this is a map that firebase would give me? In which case we might make this `Person.fromFirebaseMap`

Reply:
Yes it is from firebase map, updated the name.